src/components/application_manager/src/resumption/resumption_data_json.cc

```diff
@@ -336,20 +370,22 @@ ssize_t ResumptionDataJson::IsApplicationSaved(
 }
 
 Json::Value& ResumptionDataJson::GetFromSavedOrAppend(
-    const std::string& policy_app_id, const std::string& device_id) const {
+    const std::string& policy_app_id,
+    const std::string& device_id,
+    Json::Value& dictionary) const {
   using namespace app_mngr;
   LOG4CXX_AUTO_TRACE(logger_);
   sync_primitives::AutoLock autolock(resumption_lock_);
-  for (Json::Value::iterator it = GetSavedApplications().begin();
-       it != GetSavedApplications().end();
+  Json::Value& saved_apps = GetSavedApplications(dictionary);
+  for (Json::Value::iterator it = saved_apps.begin(); it != saved_apps.end();
        ++it) {
```

Comment:
maybe range-based

Reply:
@SNiukalov fixed in https://github.com/smartdevicelink/sdl_core/pull/2626/commits/f4e32226bda388a572a9edc282e74636a66ad925